charts/incubator/shlink/values.yaml

```diff
@@ -0,0 +1,45 @@
+image:
+  repository: shlinkio/shlink
+  pullPolicy: IfNotPresent
+  tag: "2.10.1"
```

Comment:
Wrong.

Please create the container in the mirror FIRST.
It's also actually required by now.

Reply:
Well it's not required, but you at least need to digest pin it AND remove the quotes.

Reply:
> Wrong.
> 
> Please create the container in the mirror FIRST. It's also actually required by now.

I know, but that way I can test without waiting a merge in containers repo. will fix once done!

Reply:
It's no problem to have one container too many there ;-)